src/Dom/Element.php

```diff
@@ -72,6 +72,9 @@ public function addInlineStyle($style)
      */
     public function setAttribute($name, $value)
     {
+        // Make sure $value is always a string and not null.
+        $value = $value ? $value : '';
```

Comment:
If someone passes `null`, shouldn't PHP yell at them for it? Converting `null` to empty string here would hide any potential issues in consumer code.

Reply:
I've tested this scenario with [following code snippet](https://3v4l.org/k1VTU) and found out that if we pass null as the function argument, then $value will be Null not an empty string,
```php
<?php

function test($a, $b = '')
{
    var_dump($a);
    var_dump($b); // $b is Null for second function call.
}

test('without b');
test('with null', null);
```